pkg/git/git.go

```diff
@@ -230,34 +231,61 @@ func (r *Repo) LatestNonPatchFinalToLatest() (start, end string, err error) {
 	return start, end, nil
 }
 
-func (r *Repo) latestNonPatchFinalVersion() (semver.Version, error) {
-	latestFinalTag := semver.Version{}
+func (r *Repo) LatestNonPatchFinalToMinor() (start, end string, err error) {
```

Comment:
@saschagrunert are you planning to include tests for `LatesteNonPatch*` methods in a subsequent PR?

Reply:
Oh yeah sure I was planning to bump the test coverage in the git package in the near future :)